docs/source/references.bib

```diff
@@ -315,6 +298,36 @@ @Article {liu-2018-first
   eprint =	 {1709.00106}
 }
 
+@InProceedings {dabov-2008-image,
+  author =	 {Kostadin Dabov and Alessandro Foi and Vladimir
+                  Katkovnik and Karen Egiazarian},
+  title =	 {Image restoration by sparse {3D} transform-domain
+                  collaborative filtering},
+  volume =	 6812,
+  booktitle =	 {Image Processing: Algorithms and Systems VI},
+  editor =	 {Jaakko T. Astola and Karen O. Egiazarian and Edward
+                  R. Dougherty},
+  organization = {International Society for Optics and Photonics},
+  publisher =	 {SPIE},
+  pages =	 {62--73},
+  year =	 2008,
+  month =	 Mar,
+  doi =		 {10.1117/12.766355}
+}
+
+@article {maggioni2012nonlocal,
```

Comment:
Consistency: entry type should start with upper case, and cite key is not in standard form. Same for new entry below.